app/services/broadcasts/welcome_notification/generator.rb

```diff
@@ -99,11 +99,24 @@ def commented_on_welcome_thread?
         Comment.where(commentable: welcome_thread, user: user).any?
       end
 
+      def authentication_broadcasts
+        Broadcast.active.where(type_of: "Welcome").where("title like '%_connect'")
+      end
+
+      def providers
+        # we filter the providers to suggest based on the following two criteria
+        # - provider is enabled
+        # - an acive provider Broadcast message exists
+        # Disabling/deactivating a provider_connect welcome message removes
+        # the provider from suggestions, while you could
```

Comment:
It seems there's something missing after "while you could"?

Reply:
and now, I think whatever thought I had left dangling there goes into admin docs once this is merged - the rest of the thought is probably "disable the auth provider - we won't suggest it" to match "disable the suggestion message, we'll never send it"